src/mumble/GlobalShortcut_linux.cpp

```diff
@@ -0,0 +1,187 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
```

Comment:
Why name the file `GlobalShortcut_linux` instead of `GlobalShortcutLinux`? Is it because for e.g. the windows version this naming scheme is used as well? In that case I'd vote of breaking the cycle of naming the files differently than the class that they define :point_up: